src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/HttpApi/ApiClientExtensions.cs

```diff
@@ -496,5 +493,28 @@ public static async Task<OperationStatusResponse> PollOperationToCompletion(this
 
             return operationResult;
         }
+
+        public static Task<OperationStatusResponse> PollOperationToRunning(this ApiClient apiClient, Uri operationUrl)
```

Comment:
`PollOperationToAfterStarted`? I don't like this name, but it better describes the what the method is trying to achieve. When I first saw this method called before looking at the implementation, I immediately thought "but will this complete if it missed the `Running` state?"; I'm glad it doesn't care about the state being `Running`.